packages/amplication-client/src/Project/ProjectListItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import InnerTabLink from "../Layout/InnerTabLink";
+import { Project } from "../models";
+
+type Props = {
+  project: Project;
+  className: string;
+  onProjectSelected: (project: Project) => void;
+};
+
+export const ProjectListItem = ({
+  project,
+  className,
+  onProjectSelected,
+}: Props) => {
+  return (
+    <InnerTabLink
+      icon="file"
+      to={`/${project.id}`}
+      className={className}
+      onClick={onProjectSelected} // if handleOnClick is needed instead of "to"
```

Comment:
This will not work as it is since InnerTabLink.onClick does not return the project as a parameter

Reply:
@abrl91 what about this?

Reply:
@yuval-hazaz I didn't finish resolving all your comments